### PR TITLE
New Addon - SpellCheck

### PR DESCRIPTION
**Pull request by DamienDennehy**
This replaces the previous FindTrust addon discussed recently.

### CODE REVIEW

File: addons/SpellCheck/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#SpellCheck
+
+This addon lists spells you haven't unlocked yet.
+
+##Usage
+
+//spellcheck whm | blm | smn | nin | brd | blu | geo | tru
+
+##Credits
+shinpad, trv, Arcon et all at
```
Comment (alphaONE2) [inline addons/SpellCheck/README.md]: Just a nit, nothing to block the pull request for, but it's actually "et al.", an abbreviation for the latin phrase "et alii" meaning "and others".

**Comment by alphaONE2** [comment]
Looks good to me.

**Comment by z16** [comment]
Looks good except for the indentation. You mix tabs and white space indentation. Please use 4 spaces for each indentation level.

Another very minor thing is that you sometimes use parentheses around conditional expressions in `if`-clauses, other times you do not. It's idiomatic to leave them out in Lua, but this is your choice.

Code wise it looks good, It's ready to pull as soon as you sort the indentation out.

**Comment by nitrous24** [comment]
You have no issues tracker on your fork @DamienDennehy but i think this will get a message to you.
https://github.com/Windower/Lua/issues/1377  Just tested this addon out and it lists all spells, even ones players don't have access to.

**Pull request merged**